src/routes/tools/extreme-precipitation/index.svelte

```diff
@@ -0,0 +1,473 @@
+      logException(err);
+      notifier.error("Error", err, 2000);
+    } finally {
+      isFetchingStore.set(false);
+    }
+  }
+
+  async function initApp() {
+    const { query } = $page;
+    // Get initial configuration (from default or from url)
+    const {
+      lat,
+      lng,
+      boundary,
+      scenario,
+      models,
+      imperial,
+      duration,
+      threshType,
+    } = getInitialConfig(query, DEFAULT_INITIAL_CONFIG);
+    // Set intial values for stores
+    scenarioStore.set(scenario);
+    modelsStore.set(models);
+    unitsStore.set({ imperial });
+    durationStore.set(+duration);
+    thresholdTypeStore.set(threshType);
+    const loc = await setInitialLocation(+lng, +lat, boundary);
+    locationStore.updateLocation(loc);
+    locationStore.updateBoundary(boundary);
+    const { params } = getQueryParams({
+      location: loc,
+      boundary: { id: boundary },
+      imperial: true,
+    });
+    const intervals = $returnPeriodStore;
+    const pct = threshType === DEFAULT_THRESHOLD_TYPE ? null : threshType;
+    const durationNum = +duration === 1 ? null : +duration;
+    const thresh = await getThreshold({
+      ...params,
+      intervals,
+      ...(durationNum && { duration: durationNum }),
+      ...(pct && { pct }),
+    });
+    thresholdStore.set(formatThresh(thresh));
+  }
+
+  onMount(() => {
+    initApp()
+      .then(() => {
+        appReady = true;
+      })
+      .catch((error) => {
+        console.error("init error", error);
+        logException(error);
+        notifier.error(
+          "Unable to Load Tool",
+          "Sorry! Something's probably wrong at our end. Try refereshing your browser. If you still see an error please contact us at support@cal-adapt.org.",
+          2000
+        );
+      });
+    window.scrollTo(0, 0);
+  });
+</script>
+
+<svelte:head>
+  <title>{tool.title}</title>
+  <link
+    href="https://api.mapbox.com/mapbox-gl-js/v2.0.1/mapbox-gl.css"
+    rel="stylesheet"
+  />
+</svelte:head>
+
+<Header
+  iconPaths="{tool.icons}"
+  title="{tool.title}"
+  description="An extreme weather event is an occurrence that is significantly different from typical weather at a specific location and time of year. Extreme precipitation events can lead to flooding, mudslides and other damaging events. In a changing climate the intensity and frequency of such events will likely change across California. This tool visualizes how climate models predict extreme precipitation events will change over the 21st century. "
+/>
+
+<ToolNavigation href="{`/tools/${tool.slug}`}" />
+
+<div id="explore-data" use:inview="{{}}" on:enter="{handleEntry}">
+  {#if appReady}
+    <ExploreData />
+  {:else}
+    <Loading />
+  {/if}
+</div>
+
+<div class="bx--grid">
+  <div id="about" use:inview="{{}}" on:enter="{handleEntry}">
+    <About
+      datasets="{datasets}"
+      on:datasetLoaded="{(e) => datasetStore.set(e.detail)}"
+    >
+      <div slot="description">
+        <p>
```

Comment:
@mukhtyar would you mind moving the about description to `content/tools/extreme-precipitation/about.md` and adding it similarly to the other tools? You don't need to format it to Markdown, you can just copy & paste the HTML.